wordpress/wp-admin/css/admin-menu-rtl.css

```
@@ -0,0 +1,856 @@
+
+.folded #collapse-button .collapse-button-label {
+	display: none;
+}
+
+#collapse-button .collapse-button-icon:after {
+	content: "\f148";
+	display: block;
+	position: relative;
+	top: 7px;
+	text-align: center;
+	font: normal 20px/1 dashicons !important;
```

Biome (2.1.2)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)